netbox_routing/models/ospf.py

```diff
@@ -48,6 +49,12 @@ class Meta:
 
     def __str__(self):
         return f'{self.name} ({self.router_id})'
+    
+    def get_areas(self):
+        instance_areas = OSPFArea.objects.filter(interfaces__instance=self).distinct("area_id")
+        return sorted(instance_areas.iterator(), key=lambda area: ipaddress.IPv4Address(
+            int(area.area_id) if area.area_id.isdigit() else area.area_id)
+        )
```

Comment:
Use `netaddr` instead of `ipaddress` please.  We use `netaddr` throughout the code elsewhere.